frontend/apps/marketing/cicd/3-app/template.yml.erb

```diff
@@ -326,6 +326,8 @@ Resources:
               Value: experience
             - Name: NEXT_PUBLIC_STAGE
               Value: !Ref EnvironmentType
+            - Name: NEXT_PUBLIC_DOUBLE_THE_DONATION_KEY
+              Value: !Ref EnvironmentType
```

Comment:
This should be the value of the API Key, pending the decision on whether it is a public "secret" or an environment var that's just hardcoded here. FYI @sureshc